foxy/distribution.yaml

```diff
@@ -356,6 +356,21 @@ repositories:
       url: https://gitlab.com/autowarefoundation/autoware.auto/autoware_auto_msgs.git
       version: master
     status: developed
+  avt_vimba_camera:
+    doc:
+      type: git
+      url: https://github.com/astuff/avt_vimba_camera.git
+      version: ros2_master
+    release:
+      tags:
+        release: release/foxy/{package}/{version}
+      url: https://github.com/astuff/avt_vimba_camera-release.git
+      version: 2001.1.0-1
```

Comment:
@icolwell-as `2001` is somewhat unusual for a major version.

Reply:
See discussion starting from here:
https://discourse.ros.org/t/ros2-transition-strategy/9154/14?u=ian_colwell
It's the best way we could think of for managing both ros1 and ros2 packages coming out of a single git repo.